db_structure.sql

```diff
@@ -49,4 +70,4 @@ CREATE TABLE FavoriteProducts (
     user_id INT,
     product_id VARCHAR(255),
     PRIMARY KEY (user_id, product_id)
-) 
+)
```

Comment:
Why is there no newline?

<img width="156" alt="image" src="https://github.com/damdiana/beauty-app/assets/11327761/437ba433-ef31-47ed-a3b0-f78e9dd9eb68">